lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -1,7 +1,7 @@
 const React = require('react');
 
 const title = "This is how you do it.";
-const body = "We are PUMPED to see your photos. Complete these simple steps and upload a photo of yourself in action. Your pics will inspire others to join the movement &ndash; so don't forget to share them!";
+const body = "We are PUMPED to see your photos. Complete these simple steps and upload a photo of yourself in action. Your pics will inspire others to join the movement \u2013 so don't forget to share them!";
```

Comment:
Oh sorry, just noticed this when the issue was closed. Why `\u2013` rather than just `–`?